packages/aragon-wrapper/src/cache/index.js

```diff
@@ -23,44 +19,36 @@ export default class Cache {
       //     os.homedir(), '.aragon', 'cache.json'
       //   )
       // )
-      adapter = new Memory()
-    } else {
-      adapter = new LocalStorage()
     }
-    this.db = low(adapter)
-
-    // Set default cache state
-    this.db.defaults({}).write()
 
     // Set up the changes observable
     this.changes = new Subject()
+
+    // Set up cache DB
+    this.db = localforage.createInstance({
+      driver: [localforage.INDEXEDDB, localforage.LOCALSTORAGE],
+      name: `localforage/${this.prefix}`
```

Comment:
It'd be really nice if the localstorage set up was the same as `lowdb`, but I imagine the chances of this being possible are quite slim.

Reply:
I'm not entirely sure I understand, localforage decide to use one of the drivers depending order. You meant that in case localstorage is selected, use the same config as lowdb?
I can dig deeper about that

Reply:
Yeah, if localstorage is used, it'd be nice if the data schema was the same (a quick a/b test between the two would show the differences). Again, I think the chances of this being possible is _really_ low and not a point to stress too much on.